### 利得行列

||1台稼働$a_1$|2台稼働$a_2$|
|---|---|---|
|好況$\theta_1$|$c(a_1, \theta_1)=300$|$c(a_2, \theta_1)=700$|
|不況$\theta_2$|$c(a_1, \theta_2)=300$|$c(a_2, \theta_2)=-300$|

### 自然の状態と予測値の同時分布$P(\theta, f)$

|||予測値$F$|>|>|
|||好況予測$f_1$|不況予測$f_2$|合計|
|---|---|---|---|---|
|自然の状態$\Theta$|好況$\theta_1$|$P(\theta_1,f_1)=0.35$|$P(\theta_1, f_2)=0.05$|$P(\theta_1)=0.4$|
||不況$\theta_2$|$P(\theta_2,f_1)=0.1$|$P(\theta_2, f_2)=0.5$|$P(\theta_2)=0.6$|
||合計|$P(f_1)=0.45$|$P(f_2)=0.55$|1|

条件付き確立$P(\theta_i|f_k)=P(\theta_i,f_k)\div P(f_k)$

In [2]:
# 好況予測が出されたときに好況となる確率
# 好況の確率 / 好況の予測が出ている確率 
0.35 / 0.45

0.7777777777777777

これは現状の混同行列で言うとこう。

|||猫と予測|猫ではないと予測|
|---|---|---|---|
|実際|猫だ|TP|FN|
||猫じゃない|FP|TN|

このとき、先ほどの条件付き確立は「猫と予測されたときに実際に猫である確率」になる。

式にすると以下。
$$P(\theta_{\text{猫である}}|f_{\text{猫と予測}}) = \frac{\text{TP}}{(\text{TP}+\text{FP})} = \text{Precision(精度・適合率)}$$

### 条件付き期待値

> 情報$f_k$が与えられたときに、何らかの選択$a_j$を行った時の期待金額$\text{EMV}(a_j|P(\theta|f_k))$は以下のように計算できる。

$$\text{EMV}(a_j|P(\theta|f_k)) = \sum_{i=1}^{\#\Theta}P(\theta_i|f_k)\cdot c(a_j, \theta_i)$$


$\text{EMV}(a_j|P(\theta|f_k))$を最大にする選択肢を$a^{\ast}_{f_k}$と表すとする。

これに先ほどの好況・不況予測を入れて確認する。

以下は【好況予測】が出たという条件にて【1台稼働】を採用した場合の条件付き期待金額。

\begin{align}
    \text{EMV}(a_1|P(\theta|f_1))&=\sum_{i=1}^{2}P(\theta_i|f_1)\cdot c(a_1,\theta_i)\\
    &=P(\theta_1|f_1)\cdot c(a_1,\theta_1)+ P(\theta_2|f_1)\cdot c(a_1, \theta_2)\\
    &= 0.778 \cdot 300 + 0.222\cdot 300\\
    &= 300

\end{align}    
